### PR TITLE
vmcli: use manifest for method execution

### LINKED ISSUE #1585: Update `run` in VM cli
It still supports only `Main(string, []interface{})` form (at least in doc).
1. Ensure it can be executed with any signature, update tests if needed.
2. Think about loading contract manifest (can be done automatically in `loadgo`).
3. Support `setip`/`jmp` to a specified instruction pointer if (2) will be hard.
4. Implement `reset` instruction which is useful when trying multiple executions.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1585 .
I've decided not to implement (3) and (4) because of small practical utility.

### CODE REVIEW

**Review by AnnaShaleva — APPROVED** [review]

File: pkg/vm/cli/cli.go [hunk 1]
```diff
@@ -97,7 +101,7 @@ var commands = []*ishell.Cmd{
 	},
 	{
 		Name: "loadgo",
-		Help: "Compile and load a Go file into the VM",
+		Help: "Compile and load a Go file with the manifest into the VM",
 		LongHelp: `Usage: loadgo <file>
```
[the remark this answers is not in the document]
Comment (fyrchik) [reply pkg/vm/cli/cli.go:105]: We can, however there is no need: `Events`, `SupportedStandards` and `SafeMethods` have little to no value during vmcli execution.

File: pkg/vm/cli/cli.go [hunk 2]
```diff
@@ -115,10 +119,11 @@ var commands = []*ishell.Cmd{
 	{
 		Name: "run",
 		Help: "Execute the current loaded script",
-		LongHelp: `Usage: run [<operation> [<parameter>...]]
+		LongHelp: `Usage: run [<method> [<parameter>...]]
 
-<operation> is an operation name, passed as a first parameter to Main() (and it
-        can't be 'help' at the moment)
+<method> is an contract method name, specified in manifest (and it
```
Comment (roman-khimov) [inline pkg/vm/cli/cli.go:124]: `is a contract method`

File: pkg/vm/cli/cli.go [hunk 3]
```diff
@@ -115,10 +119,11 @@ var commands = []*ishell.Cmd{
...
+        can't be 'help' at the moment). It can be '_' which will push parameters
+        on stack and execute from the current offset.
```
Comment (roman-khimov) [inline pkg/vm/cli/cli.go:126]: `onto the stack`?

File: pkg/vm/cli/cli.go (hunk 2 above)
Comment (fyrchik) [reply pkg/vm/cli/cli.go:124]: Fixed.

File: pkg/vm/cli/cli.go (hunk 3 above)
Comment (fyrchik) [reply pkg/vm/cli/cli.go:126]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**